dcc-portal-ui/app/styles/styles.scss

```diff
@@ -411,6 +412,9 @@ a.no-style {
   }
   i {
     height: 1em;
+    display: flex;
```

Comment:
IE10

Previous issues might have been due to not using vendor prefixes to support IE10 and old safari. That is now taken care of by Autoprefixer